### PR TITLE
updating abstract

### PR DESCRIPTION
**Pull request by tsurdilo**
This change is to make the abstract section of the spec much clearer than before.

### CODE REVIEW

File: workflow/spec/spec.md [hunk 1]
```diff
@@ -2,20 +2,31 @@
...
-in branches depending on different event triggers), etc.
-
-Workflows have become key components of serverless applications as they excel at orchestration and coordination
-of their functional flow. 
-
-The goal of the Serverless Workflow sub-group is to come up with a standard way for users to specify their serverless application workflow, as well as help facilitate 
-portability of serverless applications across different vendor platforms.
-
-Serverless Workflow is a vendor-neutral and portable specification which meets these goals.
+Serverless applications are becoming increasingly complex and are rearely sequential in nature.
```
Comment (cathyhongzhang) [inline workflow/spec/spec.md]: typo: change to rarely

Comment (cathyhongzhang) [inline workflow/spec/spec.md]: Nowadays people is familiar with lambda-like serverless function, which is a single function triggered by a single event. So how about changing to" Serverless applications are becoming increasingly complex and are rarely a single function that is triggered by a single event.

File: workflow/spec/spec.md [hunk 2]
```diff
@@ -2,20 +2,31 @@
...
-Workflows have become key components of serverless applications as they excel at orchestration and coordination
-of their functional flow. 
-
-The goal of the Serverless Workflow sub-group is to come up with a standard way for users to specify their serverless application workflow, as well as help facilitate 
-portability of serverless applications across different vendor platforms.
-
-Serverless Workflow is a vendor-neutral and portable specification which meets these goals.
+Serverless applications are becoming increasingly complex and are rearely sequential in nature. 
+Often they contain complex business logic to coordinate, manage, and define the execution order of a large amount 
+of serverless functions and events that can trigger those functions.
```
Comment (cathyhongzhang) [inline workflow/spec/spec.md]: Suggest to change to "Often it is complex to manually coordinate, manage ..."

File: workflow/spec/spec.md [hunk 3]
```diff
@@ -2,20 +2,31 @@
...
+ 
+Workflows have become a key component of serverless applications as they excel at dealing with cross-cutting concerns 
+such as orchestration and coordination of serverless applications functional flow.
```
Comment (cathyhongzhang) [inline workflow/spec/spec.md]: suggest to change to "as they automate the orchestration and ..."

File: workflow/spec/spec.md [hunk 4]
```diff
@@ -2,20 +2,31 @@
...
+Some of many benefits using workflows in serverless applications include:
+- Reduce complex execution logic out of your serverless functions
+- Externalize cross-cutting concerts such as parallel execution, branching, timeouts, compensation, and other flow control
+ logic out of many serverless function implementations into a single workflow definition.
+- Allow you to write less code in your serverless apps/functions
+- Significantly reduce the amount of time and effort to make changes in large serverless apps.
+
+Many different workflow implementations (both proprietary and open-source) exist today, each with it's own set of features
+and capabilities. When picking a current implementations, it is very difficult to later on switch to a different one without
+running into large time and cost investments.
```
Comment (cathyhongzhang) [inline workflow/spec/spec.md]: Suggest to change to "When you choose one implementation, it is very difficult to later on switch to a different one without investing a lot of time and cost"

File: workflow/spec/spec.md [hunk 5]
```diff
@@ -2,20 +2,31 @@
...
+Serverless applications are becoming increasingly complex and are rearely sequential in nature. 
+Often they contain complex business logic to coordinate, manage, and define the execution order of a large amount 
+of serverless functions and events that can trigger those functions.
+ 
+Workflows have become a key component of serverless applications as they excel at dealing with cross-cutting concerns 
+such as orchestration and coordination of serverless applications functional flow. 
+
+Some of many benefits using workflows in serverless applications include:
+- Reduce complex execution logic out of your serverless functions
+- Externalize cross-cutting concerts such as parallel execution, branching, timeouts, compensation, and other flow control
```
Comment (cathyhongzhang) [inline workflow/spec/spec.md]: - Allow you to develop new application much faster by taking the complex execution logic out of your application. 
- Externalize workflow execution logic and management such as ...

File: workflow/spec/spec.md (hunk 2 above)
Comment (tsurdilo) [reply workflow/spec/spec.md]: Not sure I get this. "Often they" refers to serverless applications described in the first sentence.

File: workflow/spec/spec.md (hunk 1 above)
Comment (tsurdilo) [reply workflow/spec/spec.md]: updated

File: workflow/spec/spec.md (hunk 3 above)
Comment (tsurdilo) [reply workflow/spec/spec.md]: updated

File: workflow/spec/spec.md (hunk 4 above)
Comment (tsurdilo) [reply workflow/spec/spec.md]: updated

File: workflow/spec/spec.md (hunk 5 above)
Comment (tsurdilo) [reply workflow/spec/spec.md]: updated

**Comment by tsurdilo** [comment]
@cathyhongzhang thanks for comments! updated

File: workflow/spec/spec.md (hunk 1 above)
Comment (tsurdilo) [reply workflow/spec/spec.md]: updated

**Comment by cathyhongzhang** [comment]
LGTM

**Pull request merged**